src/cli/api/local_api.py

```diff
@@ -1,21 +1,27 @@
+import os
+import shutil
+
 from typing import List, Optional, Tuple
 from pathlib import Path
-import shutil
 from cli import utils
-
+from cli.validators import validate_json
 from cli.managers.config_manager import ConfigManager
 from cli.managers.roundset_manager import RoundSetManager
 from cli.output_converter import OutputConverter
 from cli.type_aliases import (AppConfigType, ApplicationType, app_configNetworkType,
                               app_configApplicationType, assetApplicationType, assetNetworkType,
                               ExperimentType, ResultType)
 from cli.utils import read_json_file, write_json_file
-from cli.exceptions import ApplicationAlreadyExists, NoNetworkAvailable
+from cli.exceptions import ApplicationAlreadyExists, NoNetworkAvailable, ApplicationDirectoryNotComplete, \
+                           ApplicationConfigNotComplete, ApplicationSourceFilesIncomplete
+from cli.settings import BASE_DIR
 
 
 class LocalApi:
     def __init__(self, config_manager: ConfigManager) -> None:
         self.__config_manager = config_manager
+        self.app_config = Path.cwd() / "config"
```

Comment:
This is not true. app_config is not the correct directory when you do for example application create. You don't know upfront when localapi is instantiated.